Source/MQTTnet/Server/Status/MqttSessionStatus.cs

```diff
@@ -35,11 +35,5 @@ public Task DeleteAsync()
         {
             return _sessionsManager.DeleteSessionAsync(ClientId);
         }
-
-        public Task ClearPendingApplicationMessagesAsync()
```

Comment:
Why is this API removed?

Reply:
The Clear() method doesn't take into account any Qos messages in the queue, and is not used internally. Why have it?

Reply:
It is a public API. So it is correct that it is not used internally.
I agree that it has a bug in terms of QoS but I want to avoid breaking changes in the public API. This API might be used by someone even it it is not correctly implemented and the users are fine with it.